examples/fedopt_example/client.py

```diff
@@ -55,7 +55,7 @@ def train(
             # Report some batch loss statistics every so often to track decrease
             if batch_index % 100 == 0:
                 log(INFO, f"Batch Index {batch_index} of {n_batches}, Batch loss: {loss.item()}")
-            epoch_metrics.update_performance(predicted, labels)
+            epoch_metrics.update_performance(predicted, labels)  # type: ignore[arg-type]
```

Comment:
This issue should be fixed with the merging of a few other PRs. So if you merge in the latest changes from main, you should be able to get rid of these type ignores now.